docs/api-opentelemetry.asciidoc

```diff
@@ -0,0 +1,121 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/dotnet[elastic.co]
+endif::[]
+
+[[opentelemetry-bridge]]
+== OpenTelemetry Bridge
+
+NOTE: Added as experimental in v1.13.0.
+To enable it, set <<config-enable-open-telemetry-bridge, `OpentelemetryBridgeEnabled`>> to `true`.
+
+NOTE: The OpenTelemetry bridge is supported on .NET 5 and newer versions.
+
```

Comment:
I'd prefer to use our built-in `preview::[]` as it includes legal jargon about how we support technical preview features. What about:

```suggestion
preview::[]

NOTE: This feature was added in v1.13.0. It is supported on .NET 5 and newer versions.

```